javascript-big-practice/src/data/data.json

```diff
@@ -0,0 +1,28 @@
+{
+  "users": [
+    {
+      "email": "a@a.com",
+      "password": "10101010",
+      "id": 1,
+      "name": "Saul Goodman",
```

Comment:
There should be two array data:
- users (id, email, password) for authentication (login/logout)
- employees (id, name, age, phone, email, and address) for employee management features